hw03_Nyabutoa.py

```diff
@@ -0,0 +1,67 @@
+
+#################################################################################
+# Author: Alpha Nyabutop
+# Username: Nyabutoa
+#
+# Assignment:HW03: Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose:
+# Google Doc Link:https://docs.google.com/document/d/1A_q6VWbkQ8u_noD3prdv25tmySfbxk-JVBYkq4vUlPI/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+
+import turtle
+
+
+def circlec(nyabutoa):
+    nyabutoa.goto(0,0)
+    r=180
+    nyabutoa.pensize(10)
+    nyabutoa.circle(r)
+    nyabutoa.left(90)
+    nyabutoa.penup()
+    nyabutoa.forward(r+50)
+    nyabutoa.left(90)
+    nyabutoa.forward(100)
+    nyabutoa.pendown()
+    eyez(nyabutoa)
+    nyabutoa.right(90)
+    nyabutoa.penup()
+    nyabutoa.forward(20)
+    nyabutoa.right(90)
+    nyabutoa.forward(200)
+    nyabutoa.left(180)
+    nyabutoa.pendown()
+    eyez(nyabutoa)
+
+
+
+def eyez(eye1):
```

Comment:
When other developers (or code reviewers) don't understand your variable and function names, they are named poorly. Choose names that have meaning... what does `eyez` mean? similarly, what is the variable `e_r` below?